src/views/HrReports/ExperienceReports/ExperienceReport .jsx

```diff
@@ -1,142 +1,120 @@
+            // resizable: false,
+            width: 30,
+        },
+        { headerName: 'ID', field: 'em_no' },
+        { headerName: 'Name ', field: 'em_name' },
+        { headerName: 'Age ', field: 'em_age_year' },
+        { headerName: 'Mobile ', field: 'em_mobile' },
+        { headerName: 'Blood_group ', field: 'group_name' },
+        { headerName: 'Religion', field: 'relg_name' },
+        { headerName: 'Dept Name ', field: 'dept_name' },
+        { headerName: 'Dept Section ', field: 'sect_name' },
+        { headerName: 'Branch ', field: 'branch_name' },
+        { headerName: 'Technical/Non technical ', field: 'inst_emp_type' },
+        { headerName: 'Designation ', field: 'desg_name' },
+        { headerName: 'Date of joining ', field: 'em_doj' },
+        { headerName: 'Category ', field: 'ecat_name' },
+    ])
+
     return (
         <Fragment>
-            <PageLayoutCloseOnly
-                heading="Employee Experience Report"
-                redirect={RedirectToProfilePage}
-            >
-                <div className="col-md-12">
-                    <div className="row g-1">
-                        <div className="col-md-3">
-                            <DepartmentSelect style={SELECT_CMP_STYLE} onChange={handleChange} />
-                        </div>
-                        <div className="col-md-3">
-                            <DeptSectionSelect style={SELECT_CMP_STYLE} onChange={handleChangeSect} dept={dept} />
-                        </div>
-                        <div className="col-md-3">
-                            <EmpNameSelect style={SELECT_CMP_STYLE} onChange={handleChangeName} sect={sect} />
-                        </div>
-                        <div className="col-md-1">
-                            <IconButton
-                                aria-label="add"
-                                style={{ padding: '0rem' }}
-                                onClick={getEmployeeDepartment}
-                                sx={{ color: "#37575f" }}
-                            >
-                                <SearchIcon className="text-info" size={30} />
-                            </IconButton>
-                        </div>
-                    </div>
-                    <div className="col-md-12">
-                        {
-                            data === 1 ? <EmployeeExperienceTable
-                                tableData={tableData}
-                            /> : null
-                        }
-                    </div>
-                </div>
-            </PageLayoutCloseOnly>
+            <ToastContainer />
+            <CustomReport
+                columnDefs={columnDefs}
+                tableData={empDepartment}
+                onSelectionChanged={onSelectionChanged}
+                onClick={getEmployeeDepartment}
+
+                columnDefMain={columnDefMain}
+                tableDataMain={TableData}
+            />
         </Fragment>
     )
 }
```

Comment:
memo not included

Reply:
Page is not completed bcz combine all pull request. after that it will complete